### PR TITLE
[tr] Add Turkish localization for cloud native tech

### PR DESCRIPTION
**Pull request by symys**
### Describe your changes
Localize "Cloud Native Tech" page into Turkish

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Review by halil-bugol — DISMISSED** [review]
lgtm

**Review by halil-bugol — COMMENTED** [review]
DCO aktif görünmüyor kontrol eder misin @symys PR'ı imzalaman gerekiyor.

**Comment by symys** [comment]
I fixed the DCO issue, can you check again?

**Review by aliok — APPROVED** [review]
Thanks! LGTM!

**Comment by aliok** [comment]
Preview looks fine: https://deploy-preview-2344--cncfglossary.netlify.app/tr/cloud-native-tech/

File: content/tr/cloud-native-tech.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+tags: ["Fundamental", "", ""]
+---
+
+## Nedir
+
+Cloud native yığını olarak da adlandırılan cloud native teknolojileri, cloud native uygulamaları oluşturmak için kullanılan teknolojilerdir.
+Bu teknolojiler kuruluşlar için genel, özel ve hibrit bulut ortamları gibi modern ve dinamik ortamlarda ölçeklenebilir uygulamalar oluşturmaya ve çalıştırmaya olanak tanırken [bulut bilişimin](/tr/cloud-computing/) yararlarını en üst düzeye çıkarırlar.
+Bulut bilişimin yeteneklerinden yararlanmak için sıfırdan tasarlanmışlardır ve konteynerler, servis ağları, mikro servisler ve sabit altyapı bu yaklaşımın örnekleridir.
+
+## Hangi sorunu çözer
```
Comment (aliok) [inline content/tr/cloud-native-tech.md:14]:
```suggestion
## Hangi Sorunları Çözer
```

We used this header in https://deploy-preview-2344--cncfglossary.netlify.app/tr/cloud-computing/.

Let's use the same for consistency.

File: content/tr/cloud-native-tech.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+Cloud native yığını, zorlukların çeşidini adresleyen birçok farklı teknoloji kategorisine sahiptir.
+[CNCF Cloud Native Landscape’e](https://landscape.cncf.io/) baktığınızda ne kadar farklı alana temas ettiğini görebilirsiniz.
+Ancak genel olarak bahsedecek olursak cloud native teknolojileri, bir ana zorluk kümesini ele almaktadır; geleneksel BT işletim modellerinin olumsuz yönleri.
+Bu zorluk kümesi, ölçeklenebilir, arızalanırsa bile çalışmaya devam edebilen, kendi kendini onaran uygulamalar oluşturma güçlüklerinin yanı sıra kaynakların verimsiz kullanımını içermektedir.
+
+## Sorunu nasıl çözer
```
Comment (aliok) [inline content/tr/cloud-native-tech.md:21]:
```suggestion
## Nasıl Yardımcı Olur
```

We used this header in https://deploy-preview-2344--cncfglossary.netlify.app/tr/cloud-computing/.

Let's use the same for consistency.

**Review by aliok — CHANGES_REQUESTED** [review]
Just noticed some things that I would prefer to have changed.

**Review by aliok — APPROVED** [review]
LGTM!

**Review by halil-bugol — APPROVED** [review]
lgtm

**Pull request merged**